packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -97,6 +96,28 @@ class PostPublishPanel extends Component {
 						</PostPublishPanelPostpublish>
 					) }
 				</div>
+
+				<div className="editor-post-publish-panel__action-buttons">
+					{ ! submitted && (
+						<div className="editor-post-publish-panel__header-publish-button">
+							<PostPublishButton focusOnMount={ true } onSubmit={ this.onSubmit } forceIsDirty={ forceIsDirty } forceIsSaving={ forceIsSaving } />
+						</div>
+					) }
+					{ submitted && (
+						<div className="editor-post-publish-panel__header-published">
+							{ isScheduled ? __( 'Scheduled' ) : __( 'Published' ) }
+						</div>
+					) }
+				</div>
+
+				<div className="editor-post-publish-panel__disable-check">
+					<CheckboxControl
+						label={ __( 'Don’t show this again' ) }
```

Comment:
"Don’t show this panel again" might be better; I'm often wary about an unclear "this".